preflight-checks/cuda-validation/cuda_validation.py

```
@@ -0,0 +1,169 @@
+        del tensor
+        torch.cuda.empty_cache()
+        return GPUResult(index=index, name=name, passed=True, detail="ok")
+    except Exception as exc:  # noqa: BLE001 - intentional broad catch
+        return GPUResult(index=index, name=name, passed=False, detail=str(exc))
+
+
+def main() -> int:
+    log = _setup_logging()
+
+    try:
+        import torch  # noqa: WPS433 - imported here so missing torch maps to EXIT_ENV_FAIL
```

Ruff (0.15.15)

[warning] Invalid rule code in `# noqa`: WPS433

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)